Gemfile

```diff
@@ -26,7 +26,12 @@ group :development, :test do
   gem 'rake'
   gem 'rdoc'
   gem 'rspec', '~> 3.0'
-  gem 'rubocop', '~> 0.60.0', require: false
+  # TODO: change back after 2.6.0 release and updated rubocop version
```

Comment:
I wouldn't do this but 🤷‍♂️ :)

Reply:
… or adding it to `allow_failures`

Reply:
I meant I lock my rubocops. I like my builds to not fail in the future :)